examples/core/core_input_gamepad.nim

```diff
@@ -0,0 +1,179 @@
+#[
+*
+*   raylib [core] example - Gamepad input
+*
+*   NOTE: This example requires a Gamepad connected to the system
+*         raylib is configured to work with the following gamepads:
+*                - Xbox 360 Controller (Xbox 360, Xbox One)
+*                - PLAYSTATION(R)3 Controller
+*         Check raylib.h for buttons configuration
+*
+*   This example has been created using raylib 2.5 (www.raylib.com)
+*   raylib is licensed under an unmodified zlib/libpng license (View raylib.h for details)
+*
+*   Copyright (c) 2013-2019 Ramon Santamaria (@raysan5)
+*   Adapted in 2021 by jseb
+*
+********************************************************************************************]#
+
+import nimraylib_now
+
+# NOTE: Gamepad name ID depends on drivers and OS
+when defined(PLATFORM_RPI):
+  const XBOX360_NAME_ID = "Microsoft X-Box 360 pad"
+else:
+  const XBOX360_NAME_ID = "Xbox 360 Controller"
+
+const
+  PS3_NAME_ID = "PLAYSTATION(R)3 Controller"
+  XBOX360_LEGACY_NAME_ID = "Xbox Controller"
+  screenWidth = 800
+  screenHeight = 450
+
+type tex_names = enum PS3Pad, XboxPad
+var texs:array[tex_names,Texture2D]
+
+
+proc init() =
+  setConfigFlags(MSAA_4X_HINT)  # Set MSAA 4X hint before windows creation
+
+  initWindow(screenWidth, screenHeight, "raylib [core] example - gamepad input")
+
+  texs[Ps3Pad] = loadTexture("resources/ps3.png")
+  texs[XboxPad] = loadTexture("resources/xbox.png")
+
+  setTargetFPS( 60)
```

Comment:
```suggestion
  setTargetFPS(60)
```
I think there are similar style fixes for other proc calls

Reply:
Yes it's my style , but for a public sample i should either remove the parenthesis or the space.

BTW, i see in one of your sample you had written `setTargetFPS` the UFCS way :)

Reply:
There are some inconsistencies in the tests, yes. I think in Nim both UFCS and parens style are okay, even without parens has its place. But your style is something I see for the first time, I thought it was some auto-formatting issue